packages/tasit-action/src/TasitAction.test.js

```diff
@@ -353,20 +398,27 @@ describe("TasitAction.Contract", () => {
         errorFn();
       };
 
+      txSubscription = simpleStorage.setValue("hello world");
+
       txSubscription.on("error", errorListener);
 
+      txSubscription.on("confirmation", confirmationListener);
+
+      await mineBlocks(provider, 5);
+
       const snapshotId = await createSnapshot(provider);
 
-      await mineBlocks(provider, 15);
+      await mineBlocks(provider, 20);
 
       expect(confirmationFn.called).to.be.true;
 
       await revertFromSnapshot(provider, snapshotId);
 
-      await mineBlocks(provider, 20);
+      // Note: Without that, ethers.provider will keep the same
+      // receipt.confirmations as before snapshot reversion
```

Comment:
Interesting! Do you think this is expected behavior on their end?

Reply:
I'm not sure. But anyway I think that could be useful to share that on this issue: https://github.com/ethers-io/ethers.js/issues/385
What do you think?

Reply:
Yeah, great idea!

Reply:
https://github.com/ethers-io/ethers.js/issues/385#issuecomment-455187735

Reply:
Thanks! Resolving this.